aspnetcore/mvc/views/view-compilation.md

```diff
@@ -160,7 +160,7 @@ A Razor file is compiled at runtime, when the associated Razor Page or MVC view
 
 ## Razor compilation
 
-Build- and publish-time compilation of Razor files is enabled by default by the Razor SDK. Editing Razor files after they're updated is supported at build time. By default, only the compiled *Views.dll* and no *.cshtml* files or references assemblies required to compile Razor files are deployed with your app.
+Build-time and publish-time compilation of Razor files is enabled by default by the Razor SDK. Editing Razor files after they're updated is supported at build time. By default, only the compiled *Views.dll* and no *.cshtml* files or references assemblies required to compile Razor files are deployed with your app.
```

Comment:
```suggestion
Build-time and publish-time compilation of Razor files is enabled by default by the Razor SDK. Editing Razor files after they're updated is supported at build time. By default, only the compiled *Views.dll* and no *.cshtml* files or references assemblies required to compile Razor files are deployed with an app.
```

Reply:
Hold off on merging this suggestion ... I'll do it manually on my commit.

Reply:
Oooops! Other file ...... so much for cooking dinner 👨‍🍳 and working on the repo!

Reply:
Did you mean  🍻  with dinner 🤣